<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1101</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="AJOUTER">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>1091</width>
        <height>491</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>add</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>-9</y>
          <width>771</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_AJOUTER">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>AJOUTER</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>110</y>
          <width>191</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="le_ID_EMPLOYES"/>
         </item>
         <item>
          <widget class="QLineEdit" name="le_NOM"/>
         </item>
         <item>
          <widget class="QLineEdit" name="le_PRENOM"/>
         </item>
         <item>
          <widget class="QLineEdit" name="le_ADRESSE_EMAIL"/>
         </item>
         <item>
          <widget class="QLineEdit" name="le_ADRESSE"/>
         </item>
         <item>
          <widget class="QLineEdit" name="le_NUM_TEL"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>121</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ID_EMPLOYES</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>NOM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>PRENOM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>ADRESSE_EMAIL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>ADRESSE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>NUM_TEL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>mod/supp</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>791</width>
          <height>251</height>
         </rect>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>PRENOM</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>MODIFIER</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ADRESSE_EMAIL</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>290</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_EMPLOYES</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>NUM_TEL</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ADRESSE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>160</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="mod_NUM_TEL"/>
         </item>
         <item>
          <widget class="QLineEdit" name="mod_NOM"/>
         </item>
         <item>
          <widget class="QLineEdit" name="mod_PRENOM"/>
         </item>
         <item>
          <widget class="QLineEdit" name="mod_ADRESSE_EMAIL"/>
         </item>
         <item>
          <widget class="QLineEdit" name="mod_ADRESSE"/>
         </item>
         <item>
          <widget class="QLineEdit" name="mod_ID_EMPLOYES"/>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>-240</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>290</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>pdf</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>301</x>
          <y>261</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Par Default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ID_EMPLOYES</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NOM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PRENOM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ADRESSE_EMAIL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ADRESSE</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>chat boot</string>
       </attribute>
       <widget class="QPlainTextEdit" name="conv">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>391</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="msgboot">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sendboot">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>290</y>
          <width>93</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>send</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>mailling</string>
       </attribute>
       <widget class="QTextEdit" name="msg">
        <property name="geometry">
         <rect>
          <x>148</x>
          <y>126</y>
          <width>651</width>
          <height>291</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>type your message here ...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="browseBtn">
        <property name="geometry">
         <rect>
          <x>725</x>
          <y>89</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rcpt">
        <property name="geometry">
         <rect>
          <x>148</x>
          <y>20</y>
          <width>491</width>
          <height>21</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>recipient</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="subject">
        <property name="geometry">
         <rect>
          <x>148</x>
          <y>54</y>
          <width>651</width>
          <height>16</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>subject</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendBtn">
        <property name="geometry">
         <rect>
          <x>148</x>
          <y>420</y>
          <width>651</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="file">
        <property name="geometry">
         <rect>
          <x>148</x>
          <y>90</y>
          <width>571</width>
          <height>23</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>attachement here</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="mail_pass">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>20</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>your mail password</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>60</y>
          <width>181</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="idE"/>
         </item>
         <item>
          <widget class="QLineEdit" name="code"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Ouvrir</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="le_mail">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_motdepasse">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="login">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
